src/plays/dev-jokes/devJokes.css

```diff
@@ -0,0 +1,76 @@
+.maincontanier {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  align-items: center;
+  background-image: url("background.jpg");
+  background-position: center;
+  background-size: cover;
+  height: 100%;
+  width: 100%;
+}
+.jokecontanier {
+  margin: 20px;
+  padding: 20px;
+  width: 600px;
+  min-height: 400px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border-radius: 10px;
+  box-shadow: 0px 6px 10px rgba(0, 0, 0, 0.25);
+  transition: all 0.2s;
+  margin-top: 30px;
+  background-color: rgb(0, 0, 0);
+  background-color: rgba(199, 183, 183, 0.4);
+  font-size: x-large;
+}
+
+.jokecontanier:hover {
+  box-shadow: 0px 6px 10px rgba(0, 0, 0, 0.4);
+  transform: scale(1.01);
+}
+
+.btn {
```

Comment:
Please use play-specific naming and avoid standard naming conventions just to avoid a collision.

something like `dev-jokes-btn`